src/app/modules/user/user.controller.ts

```diff
@@ -35,31 +38,73 @@ export const handleContactSendOtp: RequestHandler<
   // Guard against user updating for a different user, or if user is not logged
   // in.
   if (!sessionUserId || sessionUserId !== userId) {
-    return res.status(StatusCodes.UNAUTHORIZED).send('User is unauthorized.')
+    return res
+      .status(StatusCodes.UNAUTHORIZED)
+      .json({ message: 'User is unauthorized.' })
```

Comment:
this is not a backwards-compatible change right? what will `edit-contact-number-modal.client.controller.js` show when old clients error?